pages/your_domain/health_data.md

```diff
@@ -0,0 +1,140 @@
+
+EHDS places a significant emphasis on promoting the secondary use of health data, to serve research, innovation, policy making and regulatory purposes. To achieve this, several European initiatives have been launched to establish its foundation and define guiding principles. Among these endeavors, two signification projects stand out:
+- [TEHDAS]( https://tehdas.eu/), the joint action Towards the European Health Data Space, helped develop and promote concepts for the secondary use of health data to benefit public health and health research and innovation in Europe. TEHDAS produced [a set of recommendations]( https://tehdas.eu/results/) for the European Commission and member states to enable secondary use of health data.
+- [HealthData@EU Pilot](https://ehds2pilot.eu/) is a project designed to build a pilot version of the EHDS infrastructure for the secondary use of health data “HealthData@EU”. The project establishes connections between data platforms within a network infrastructure and develops services that facilitate the user journey for research projects using health data. It also offers guidelines for data standards, data quality, data security and data transfer to support this cross-border infrastructure.
+
+The EU published an overview of the [national laws on electronic health records](https://health.ec.europa.eu/other-pages/basic-page/overview-national-laws-electronic-health-records-eu-member-states-2016_en) in 2016. Under US law, EHR data is classified as protected health information (PHI) and must comply with security laws and encrypt data properly when sending or retrieving such information. Examples of protected health data include patient names, phone numbers, addresses, dates of birth, social security numbers, and insurance information.
+  * Regulations: [Health Insurance Portability and Accountability Act (HIPAA) and Health Information Technology for Economic and Clinical Health (HITECH)](https://www.hhs.gov/hipaa/for-professionals/security/laws-regulations/index.html)
+
+The [International Patient Summary (IPS)](https://international-patient-summary.net/) is a standardised set of basic patient-related physiological and clinical data that includes the most important health and care related facts required to ensure safe and secure healthcare. It comprises data about medications, allergies/intolerances, problems, immunizations, results and procedures for a specified patient. It is a joint standard of five standard development organisations (CEN, HL7, IHE, ISO and SNOMED) and is actively supported by the Global Digital Health Partnership and the World Health Organisation.
+
+### Solutions
+
+It is likely that the use of one of the following standards will not be a choice, but something imposed by the software providers of the EHR systems.These interchange formats are the standards that are likely to be present as part of the EHR, and include:
```

Comment:
```suggestion
Likely, the use of one of the following standards will not be a choice, but something imposed by the software providers of the EHR systems. These interchange formats are the standards that are likely to be present as part of the EHR, and include:
```

Reply:
I would prefer to keep the original text.

Reply:
I have edited the suggestion above.